### DISCUSSION TITLE
block edition  in workflow？？

### DISCUSSION THREAD

**Discussion by Jianbin719** [YetiForceCompany/YetiForceCRM#15707] [How to]
How to unlock record after block edition triggered in workflow？？

**Comment by rskrzypczak** [comment]
Solution:
Disable the edition lock in workflow :)

If you need to lock edition of a record and a status field is enough for you, I would recommend using the record lock functionality available in the picklist configuration.  
A record locked using this method will be possible to unlock directly from the record. Of course only of an action from the profile settings allows that.
![image](https://user-images.githubusercontent.com/10221999/153607600-a2539b6f-c47a-4cf9-ab50-ab26e3d1124d.png)

**Reply by Jianbin719** [reply]
Thanks for your recommendation
